openapi.yaml

```diff
@@ -0,0 +1,700 @@
+# This is the Open API interface for Open Service Broker API. Every attempt will
+# be made to make the Open API version of OSB API accurately represent the
+# written specification. If the spec and this document conflict, the spec is
+# the authority.
+
+openapi: "3.0.0"
+
+info:
+  title: Open Service Broker API
+  description: The Open Service Broker API defines an HTTP(S) interface between Platforms and Service Brokers.
+  license: 
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact: 
+    name: Open Service Broker API
+    url: https://www.openservicebrokerapi.org/
+    email: open-service-broker-api@googlegroups.com
+  version: v2.13
+
+servers:
+  - url: http://example.com
+    description: Broker Endpoint
+  - url: http://localhost:3000
+    description: OSB Checker - MockOSB
+
+paths:
+  /v2/catalog:
+    get:
+      summary: get the catalog of services that the service broker offers
```

Comment:
Can we start these with capital letters?  Get the catalog...
It looks better to me when generated.

Reply:
We can in a follow-up!